snippets/gift-card-recipient-form.liquid

```diff
@@ -181,18 +178,60 @@
           {% render 'icon-error' %}
           <span class="error-message">
             {%- if form.errors contains 'message' -%}
-              {{ form.errors.translated_fields.message | capitalize }}
               {{ form.errors.messages.message }}.
             {%- endif -%}
           </span>
         </div>
       </div>
+
+      <div class="recipient-fields__field">
+        <div class="field">
+          <input
+            class="field__input text-body"
+            autocomplete="send_on"
+            type="date"
+            id="Recipient-send_on-{{ section.id }}"
+            name="properties[Send on]"
+            placeholder="{{ 'recipient.form.send_on' | t }}"
+            pattern="\d{4}-\d{2}-\d{2}"
+            value="{{ form.send_on }}"
+            {% if form.errors contains 'send_on' %}
+              aria-invalid="true"
+              aria-describedby="RecipientForm-send_on-error-{{ section.id }}"
+            {% endif %}
+          >
+          <label class="form__label field__label" for="Recipient-send_on-{{ section.id }}">
+            {{ 'recipient.form.send_on_label' | t }}
+          </label>
+        </div>
+
+        <div
+          id="RecipientForm-send_on-error-{{ section.id }}"
+          class="form__message{% unless form.errors contains 'send_on' %} hidden{% endunless %}"
+        >
+          {% render 'icon-error' %}
+          <span class="error-message">
+            {%- if form.errors contains 'send_on' -%}
+              {{ form.errors.messages.send_on }}.
+            {%- endif -%}
+          </span>
+        </div>
+      </div>
     </div>
     <input
       type="hidden"
       name="{{ gift_card_recipient_control_flag }}"
       value="if_present"
       id="Recipient-Control-{{ section.id }}"
     >
+    <input
+      type="hidden"
+      name="properties[__shopify_offset]"
+      value=""
+      id="Recipient-Offset-{{ section.id }}"
```

Comment:
Yep it's as you say, the difference between UTC and local time. It exists because UX wanted the user's date input to be interpreted from their perspective rather than from the shops's perspective. It means the gift card is sent on the morning of the day selected in the user's timezone. If we didn't do it then we could only guarantee +- 1 day. In terms of naming, yes it's still changeable. It's an internal detail that users of flagship themes should basically never be aware of, but it could make a difference to other theme developers.

Reply:
Great sounds good. I think we could match the name of the JS method: `getTimezoneOffset`. So maybe `Recipient-timezone-offset-{{ section.id }}`